openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1097,3 +1098,33 @@ def verify_if_more_than_n_percentage_pod_restarted(
             msg = "Out of {} pods {} pods restarted {}".format(
                 len(pods_new), len(pods_restart), pods_restart)
             raise AssertionError(msg)
+
+    def _validate_gluster_pod_memory(self, size_limit):
```

Comment:
You can also describe a little bit about the size_limit argument in the function description.